pallets/loans/src/lib.rs

```diff
@@ -549,12 +606,14 @@ impl<T: Config> Pallet<T> {
 
     pub fn update_exchange_rate(currency_id: CurrencyId) -> DispatchResult {
         // exchangeRate = (totalCash + totalBorrows - totalReserves) / totalSupply
+        let total_cash = Self::get_total_cash(currency_id);
         let total_borrows = Self::total_borrows(currency_id);
+        let total_reserves = Self::total_reserves(currency_id);
         let total_supply = Self::total_supply(currency_id);
-        let total_cash = Self::get_total_cash(currency_id);
 
         let cash_plus_borrows = total_cash
```

Comment:
`cash_plus_borrows` needs a better name.

Reply:
I haven't found a good name, could you give some better advice?

Reply:
How about `cash_plus_borrows_minus_reserves`